src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('x is %i, y is %s, and z is "%s"' % (x, y, z))
```

Comment:
Great start! See if you can take this one step further for lines 14 and 17:

`y` is a floating-point decimal `2.24552`

The requested output requires you to round `y` to precision 2. 

See if you can achieve this using format codes: https://thepythonguru.com/python-string-formatting/

Reply:
Thanks, I didn't even notice that!